src/components/movie-list/MovieList.jsx

```diff
@@ -0,0 +1,26 @@
+import "./MovieList.css";
+export const MovieList = ({ movie }) => {

```

Comment:
Small things, like having a space between import and component can improve readability 😊